<?xml version="1.0"?>
<robot name="myfirst">
    <link name="base_link"/>
    <link name="link1"/>
    <link name="link2"/>

    <joint name="joint1" type="continuous">
        <parent link="base_link"/>
        <child link="link1"/>
        <origin xyz="2 0 0" rpy="0 0 0" />
        <axis xyz="0 1 0" />
    </joint>

    <joint name="joint2" type="continuous">
        <parent link="link1"/>
        <child link="link2"/>
        <origin xyz="2 0 0" rpy="0 0 0" />
        <axis xyz="1 0 0" />
    </joint>
</robot>
